File: library/src/main/java/cn/cqray/android/dialog/amin/NoAnimator.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 无动画
 * @author Cqray
 */
public class NoAnimator extends DialogAnimator {

    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.duration(0);
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/NoAnimator.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 无动画
 * @author Cqray
 */
class NoAnimator : DialogAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.duration(0)
    }
}